### ISSUE TITLE
Shamisen Tablature support 

### ISSUE THREAD

**Issue by emlbrg** [musescore/MuseScore#25727]
### Your idea

I have noticed that so far MuseScore only supports shamisen with 5-line staff, but 3-line tab is commonly used in shamisen scored. Therefore, I think it would be a nice idea to add this to the roaster of features. As the name shamisen ("3 lines") suggests, this is a string instrument with 3 strings. Generally speaking, the strings are tuned in one of three tunings, named "Hon-cyoshi", "Ni-agari", and "San-sagari" (see picture for details)
<img width="523" alt="Screenshot 2024-12-04 at 20 13 22" src="https://github.com/user-attachments/assets/d4d1032d-0706-489a-8d48-127bfb3897c1">
There are actually 3 types of shamisen based on size, and I admit my knowledge is limited to the Futozao shamisen (the largest of the three), but the same principles apply to all, thought range may differ (I will look into this if need be). 
In shamisen tabs, each line corresponds to a string, the string number 1 being at the bottom, and in lieu of notes a number is placed on the line. These numbers correspond to positions on the neck of the instrument, as described in the figure below (ignore the middle scheme for now). This figure also indicates a sensible range for the strings, tho variation is know to happen (until the strings snap, that is).
![unnamed](https://github.com/user-attachments/assets/7d3518a4-2f6d-45b8-b68b-71edec2e85f0)
In addition, there are special engravings used for shamisen notation. As you a see from the score below, signs such are ス and ハ, as well as ligatures of various kinds and lines underneath the numbers indicating positions are used. I believe it would be possible to use most of these with the current MuseScore engravings but I am be wrong.  
![tsugaru_shami_020](https://github.com/user-attachments/assets/003621b3-c52e-4c38-8dff-9eb5c050968f)
I am not sure what other settings may be needed at this point but I am happy to elaborate further. Thank you. 

### Problem to be solved

Support for shamisen tablature. 

### Checklist

- [X] This request follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting issues
- [X] I have verified that this feature request has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar requests

**Comment by Jojo-Schmitz** [comment]
Only thing needed to make tablature work for Shamisen is string data: number of strings and their tunings and the number of frets, all in instruments.xml.
Check the handmade template from https://musescore.org/en/node/372402#comment-1270337

**Comment by MarcSabatella** [comment]
Well, from the image, also there appears to be a particular way that note values are indicated, using lines under the fret numbers.  Also there appear to be some other special symbols, maybe some sort of articulation?

**Comment by Jojo-Schmitz** [comment]
Yes, there's more to in in this issue than 'only' tablature. That'd be a first step and prerequisite though
